### PR TITLE
Config cleanup - patroni cluster health/405

### PR DESCRIPTION
**Pull request by andrea-williams**

### CODE REVIEW

File: openshift/Jenkinsfile [hunk 1]
```diff
@@ -44,6 +44,7 @@ pipeline {
                     // Env Canada Subscriber
                     sh "PROJ_TARGET=${projProd} ./openshift/scripts/oc_provision_ec_cronjob.sh prod apply"
                     sh "PROJ_TARGET=${projProd} ./openshift/scripts/oc_provision_bcfw_p1_forecasts_cronjob.sh prod apply"
+                    sh "CPU_REQUEST=100m CPU_LIMIT=2000m MEMORY_REQUEST=512Mi MEMORY_LIMIT=2Gi POD_MANAGEMENT_POLICY=OrderedReady PROJ_TARGET=${projProd} ./openshift/scripts/oc_provision_db.sh prod"
```
Comment (Sybrand) [inline openshift/Jenkinsfile:47]: This shouldn't be here - the db provision happens up top

File: openshift/templates/deploy.dc.yaml [hunk 2]
```diff
@@ -43,6 +43,8 @@ parameters:
   - name: WFWX_MAX_PAGE_SIZE
     description: Maximum number of entities to request at a time from the WFWX API
     value: "1000"
+  - name: PROJECT_NAMESPACE
+    description: Openshift project namespace. Used for /health check
```
Comment (Kyubinhan) [inline openshift/templates/deploy.dc.yaml:47]: Can we make this required?

**Comment by andrea-williams** [comment]
> I wonder why all the old commits show up in this pr, thoughts?

@Kyubinhan It's because silly ol' me forgot to pull from main after my previous PR had been merged before I started on this PR

**Comment by andrea-williams** [comment]
> Looks good - but please remove line 47 from the jenkinsfile (unless I'm missing something - I don't know how that sneaked in?)

yes I also don't know. I make a point of never touching Jenkinsfiles ever

**Pull request merged**